material/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "mkdocs-material",
```

Comment:
As I understand, everbody who will be using the theme will have these values set? Wasn't the fact that this is not configurable the reason why we rejected a native PWA integration in the first place?

Reply:
Yeah! So, I mainly put that as a placeholder for now, need to figure out a way to get the config values from mkdocs.yml into here. If that is not at all possible, then I need to rethink.

Reply:
A plugin may be a better approach than to encode this into the theme, as you would have to jump through a lot of hoops to extra information from `mkdocs.yml` in anything other than `*.html` and `*.xml` files.